schemas/participant.json

```diff
@@ -0,0 +1,137 @@
+            "valueType": "string",
+            "meta": {
+                "core": true,
+                "displayName": "Gender",
+                "notes": "Source : MOH; mCODE3: Patientgender;Phenopacket: Individual.gender; Beacon V2: Individual.gender; EDI indicator"
+            },
+            "restrictions": {
+                "codeList": "#/list/gender",
+                "required": false
+            }
+        },
+        {
+            "name": "ethnicity",
+            "description": "Refers to the ethnic or cultural origins of a person’s ancestors. The provided values are based on the list of ethnic or cultural origins 2021 defined by Statistics Canada",
+            "valueType": "string",
+            "meta": {
+                "core": true,
+                "displayName": "Ethnicity",
+                "notes": "mCODE3: Patient.extension:ethnicity; EDI indicator"
+            },
+            "restrictions": {
+                "codeList": "#/list/ethnicity",
+                "required": false
+            }
+        },
+        {
+            "name": "race",
+            "description": "A social construct used to judge and categorize people based on perceived differences in physical appearance in ways that create and maintain power differentials within social hierarchies. There is no scientifically supported biological basis for discrete racial groups. The provided values are based on race-based data standard defined by CIHI guidance ",
+            "valueType": "string",
+            "meta": {
+                "core": true,
+                "displayName": "Race",
+                "notes": "mCODE3: Patient.extension:race; EDI indicator"
+            },
+            "restrictions": {
+                "codeList": "#/list/race",
+                "required": false
+            }
+        },
+        {
+            "name": "vital_status",
+            "description": "Donor's last known state of living or deceased.",
+            "valueType": "string",
+            "meta": {
+                "core": true,
+                "displayName": "Vital Status",
+                "notes": "Source: ARGO; CQDG: vital_status; Phenopacket: vital_status.id; Beacon V2: vital_status.id"
+            },
+            "restrictions": {
+                "codeList": "#/list/vital_status",
+                "required": true
+            }
+        },
+        {
+            "name": "date_of_birth",
+            "description": "Indicate participant's date of birth.",
+            "valueType": "string",
+            "meta": {
+                "core": true,
+                "displayName": "Date Of Birth",
+                "notes": "Source: MOH; All4One : Month,year of birth; mCODE STU3: Patient.birthDate; Phenopacket: Individual.date_of_birth; Beacon V2: Individual.date_of_birth"
```

Comment:
We put `Source` in all our notes... would we want to just add a `source` property to our meta?

Similarly, would we want a separate property to list the field from other data sets that maps here?
ex. `"mapping": "All4One:Month,year of birth|mCODE STU3: Patient.birthDate|Phenopacket:Individual.date_of_birth"` etc.